### PR TITLE
feat: webpack config

### PR DESCRIPTION
**Pull request by johnvente**
Initial Webpack configuration for React app in the XBlock

### React app
react-start

## React app inside XBlock
Xblock-webpack

**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

### CODE REVIEW

File: imagesgallery/static/html/bundle.js
Comment (Ian2012) [inline imagesgallery/static/html/bundle.js:1]: Does this need to be committed to version control?

File: react-app/App.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.test-div {
```
Comment (Ian2012) [inline react-app/App.css]: Choose a more appropriate name for this class

File: imagesgallery/static/html/bundle.js
Comment (johnvente) [reply imagesgallery/static/html/bundle.js:1]: yep it does, any change should have it's bundle files

File: react-app/App.css (hunk 1 above)
Comment (johnvente) [reply react-app/App.css]: Done!

**Review by Ian2012 — APPROVED** [review]

**Pull request merged**